### PR TITLE
SurveyFormPR

### PR DESCRIPTION
**Pull request by DevKaranJ**
This pull request includes the code from my Coding partner @mumbereh Admissions Trial project, which I have copied into this new repository. I have created a new branch for this code and now I am requesting a code review.

As the Code Reviewer, I have carefully reviewed the code and made several observations and comments. I have followed the guidelines from the article "How to give a code review?" to ensure a comprehensive review.

### CODE REVIEW

**Comment by DevKaranJ** [comment]
Hi @DevKaranJ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights ✔️
 Good UI design ✅
 Good use of HTML & CSS ✅
 No linter errors ✅
 Pull request has a descriptive title and summary ✅
 Repository has a professional readme file ✅
Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        <input type="radio" value="Maybe" name="microverse">Maybe<br>
+        <input type="radio" value="Not sure" name="microverse">Not sure<br>
+        <select id="dropdown">
+            <option>Select current role</option>
+            <option>Student</option>
+            <option>Full Time Job</option>
+            <option>Full time learner</option>
+            <option>prefer not to say</option>
+            <option>other</option>
+        </select><br>
```
Comment (DevKaranJ) [inline index.html:42]: - [ Required ] I think there is some duplicated code on this page, contact **current role** Section appear twice on the same page. I believe this is not the desired outcome, please see the screenshot below and kindly rectify this.

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @DevKaranJ  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct use of GitHub inline comment feature✔
-  Review comment is phrased politely👋 ✔

 ## Required Changes ♻️
> Add as many comments as possible in the given time and finish your review. Remember to add a minimum of 2 comments - you will definitely find something that could have been done better!
- [ ] You did great. :+1:  your comment was clear and polite, But looking at the project requirement, You are required to add at least 2 review comments, I'll suggest you go through the project and add a few more reviews. :+1: :smile: 

## Optional suggestions
- [OPTIONAL] I'll suggest you try to use a couple of emojis and/or gifs in the review. they make the code review more human-friendly. :+1: :robot: :boy: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bantealem — CHANGES_REQUESTED** [review]
Status: Required Changes
Hi @DevKaranJ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Polit response 

 ## Required Changes ♻️
- [ ] As suggested by the previous reviewer, please make sure to give in-line comments on the PR. It will make your request clear and to the point. To make it just go "file changed" button on the right top part of the PR then you will get the codes after that click on the line of code that you want to give comments.
image

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+body {
+    color: whitesmoke;
```
Comment (DevKaranJ) [inline style.css]:
```suggestion
    color:  rgb(7, 7, 7);
```

File: index.html (hunk 1 above)
Comment (DevKaranJ) [inline index.html:42]: - [ Required ] I think there is some duplicated code on this page, contact current role Section appear twice on the same page. I believe this is not the desired outcome, please see the screenshot below and kindly rectify this.

-

**Review by Shisui6 — APPROVED** [review]
Hi @DevKaranJ, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Clear and polite comments ✅


## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**